packages/app/i18n/en.pot

```diff
@@ -689,3 +662,21 @@ msgstr ""
 
 msgid "Quarters per year"
 msgstr ""
+
+msgctxt ""
+"COLUMNS,\n"
+"                    visType\n"
+"                )}."
+msgid ""
+"Add at least one item to ${getAxisNameByVisType(\n"
```

Comment:
This won't work for translations, unfortunately.  The dynamic part won't be translated, first of all, and secondly the function call shouldn't show up in the translation string - you can use [interpolation](https://www.i18next.com/translation-function/interpolation) for that, but again the axis names need to also be translatable.

Reply:
Looks like the names are translated in the `@dhis2/analytics`, so using interpolation should be enough here.  It should be tested to make sure the translations are complete and correct.

Reply:
Oops, not sure why I missed the interpolation here, using it everywhere else 🙈 thanks for spotting it, fixed with the latest commit